content/en/docs/reference/command-line-tools-reference/feature-gates/external-service-account-token-signer.md

```diff
@@ -0,0 +1,14 @@
+---
+title: ExternalServiceAccountTokenSigner
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.32"
+---
+Make the kube-apiserver use [ExternalJWTSigner](/docs/reference/access-authn-authz/service-account-admin#external-token-signer) for external token signing and key management.
+Requires `ExternalServiceAccountTokenSigner` to be enabled.
```

Comment:
Drop this line, this file is documenting the feature gate itself

Reply:
Done